ClinGen/clincoded
# Connects to #326. GDM linkbacks and breadcrumbs

* Briefcase icon linking back to GDM in Curation Header
![image](https://cloud.githubusercontent.com/assets/4326866/12875103/a4897196-cd97-11e5-8e53-58ec8d2ed061.png)
* Briefcase icon in submit page 'Return to Curation Central' buttons
![image](https://cloud.githubusercontent.com/assets/4326866/12875110/e592e33e-cd97-11e5-8303-c63ebc79145a.png)
* Remove 'View' buttons from submit pages; rename 'Edit' button in certain pages to 'Edit/Assess' 
![image](https://cloud.githubusercontent.com/assets/4326866/12875117/18c5fa16-cd98-11e5-9c46-4de7920e011d.png)
* Hierarchy breadcrumb Edit and View pages:
![image](https://cloud.githubusercontent.com/assets/4326866/12875142/814d7d70-cd98-11e5-806c-2a8163437194.png)
* Changes to calculated properties

Testing:

1. Create GDM, add PMID
3. Confirm that the Briefcase icon in the Header is visible, but not a link
3. Add all sorts of data types
  * On Submit pages, confirm that the Return to Curation Central buttons have the briefcase icon, the View button is not there, and the Edit is Edit/Assess where applicable
  * Check the View and Edit pages to make sure that the GDM Linkback Breadcrumbs work, and that briefcase icon in the Header is linked properly
